src/style.css

```diff
@@ -1 +1,60 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: 100px;
+  justify-content: space-between;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--0 > * {
```

Comment:
The use of the `*` selector is not recommended as it can impact performance. It would be better to use a class selector or another more specific selector here.